### ISSUE TITLE
Add "Uploaded during the last 30 days" to UserID statistics

### ISSUE THREAD

**Issue by edickens** [FreeUKGen/MyopicVicar#2205]
These are the totals at present..
![Capture.JPG](https://images.zenhubusercontent.com/5cb4c5e2cccdd751e9e85870/7d0e93e2-c0d6-4207-b623-6fca27c34e5c)
But although it says how many have uploaded batches, some batches may have been years ago.  I'd like to know who is active now.

**Comment by SteveBiggs** [comment]
I agree in principle, but some active transcribers are quite slow so perhaps the period should be more than 30 days - say, 3 months? I know we encourage them to upload as they go along, but many do not.

**Comment by edickens** [comment]
I agree with Steve and have changed the title.
